pad.h

```diff
@@ -348,6 +348,11 @@ Restore the old pad saved into the local variable C<opad> by C<PAD_SAVE_LOCAL()>
 #define SVpad_STATE		PADNAMEf_STATE
 #define SVpad_TYPED		PADNAMEf_TYPED
 #define SVpad_OUR		PADNAMEf_OUR
+#define PADNAMEt_OUTER          PADNAMEf_OUTER
+#define PADNAMEt_STATE          PADNAMEf_STATE
+#define PADNAMEt_LVALUE         PADNAMEf_LVALUE
+#define PADNAMEt_TYPED          PADNAMEf_TYPED
+#define PADNAMEt_OUR            PADNAMEf_OUR
```

Comment:
To avoid accidental use in-core, these should be wrapped in `#ifndef PERL_CORE`.

Reply:
Huh. Wow it turns out quite a few bits of existing code in core were still even using those old names from the previous round of renaming. I'll fix those up too.

Reply:
If I were being _really_ nitpicky I would do the commits in the opposite order, but I shan't ;-)